Main/src/Scoring.cpp

```diff
@@ -1025,7 +1025,7 @@ bool Scoring::m_IsBeingHold(const ScoreTick* tick) const
 	if (!m_prevHoldHit[index]) return false;
 
 	// b) The last button release happened inside the 'near window' for the end of this hold object.
-	if (obj->time + obj->duration - m_buttonReleaseTime[index] - m_inputOffset > Scoring::goodHitTime) return false;
+	if (obj->time + obj->duration - m_buttonReleaseTime[index] - m_inputOffset > Scoring::holdHitTime) return false;
```

Comment:
This would make the last tick very lenient, to the point where it would become a free combo. The BPM would have to be as low as around 100 for this window to be relevant, otherwise the previous tick would be inside the hit window and thus you'd be able to ignore the last tick entirely.

Reply:
I just playtested Black or White GRV on konasute and on USC (w/ recreated chart), and two felt quite similar.
I could be wrong, but in specific I didn't feel that "the last tick judgement" was too relaxing.